crates/rtmp/examples/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "scuffle-rtmp-examples"
+version = "0.1.0"
+edition = "2024"
+description = "Examples for scuffle-rtmp"
+repository = "https://github.com/scufflecloud/scuffle"
+authors = ["Scuffle <opensource@scuffle.cloud>"]
+readme = "README.md"
+documentation = "https://docs.rs/scuffle-rtmp-examples"
+license = "MIT OR Apache-2.0"
+keywords = ["rtmp", "server", "streaming"]
```

Comment:
remove all of this and replace it with `publish = false` or this crate will be published to `crates.io`